rand_chacha/src/guts.rs

```diff
@@ -12,7 +12,9 @@
 use ppv_lite86::{dispatch, dispatch_light128};
 
 pub use ppv_lite86::Machine;
-use ppv_lite86::{vec128_storage, ArithOps, BitOps32, LaneWords4, MultiLane, StoreBytes, Vec4, Vec4Ext, Vector};
+use ppv_lite86::{
+    vec128_storage, ArithOps, BitOps32, LaneWords4, MultiLane, StoreBytes, Vec4, Vec4Ext, Vector,
+};
```

Comment:
Multi-line imports are a little ugly and don't work great with diffs. Best fix is to split into multiple `use` statements?

Reply:
In my experience, they are fine. Especially considering that `rand_chacha` is rarely touched.